### DISCUSSION TITLE
Suggestions for the next release (1.7.10)

### DISCUSSION THREAD

**Discussion by letovskiiy** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1343] [Q&A]
Hi, thanks for the update! I would like to ask you to add:
1. When the radar is triggered and an anti-ballistic missile takes off, you can make the sounds of a missile being shot down in the sky so that it can be heard, and to make it a little more realistic, please add more fragments after the anti-ballistic missile has been triggered.
2. Add the possibility of adding several rockets to the launch pad at once, so that the second one immediately appears after the first rocket takes off... third... the fourth and so on, at least make it possible to add 10-15 missiles to one cell for a rocket. 
3. Please add to the rocket artillery turret the ability to launch missiles at target designators (manual, short range, and so on); please make sure that artillery shells are shot down by an anti-ballistic missile in a row
4. Please create one common mechanism of the air defense system, such as a turret, but inside with a radar, so that it is not divided as a radar - launcher - radar screen with the ability to launch at least 12 missiles.
5. You can add unmanned aerial vehicles so that you can hit positions and be seen by radar.
6. I ask you to make sure that the radar sees the aircraft that are being produced for airstrikes, and there is the possibility of shooting down aircraft with anti-ballistic missiles

Thank you in advance.
P.s. I ask you to do everything so that the radar does not fail, and shoots down missiles in time. I will ask you to help me figure out what it is in the radar: smart mode, and red stone mode, how does it work?

**Comment by gmhs022** [comment]
2 is already possible somewhat, just insert missiles using a hopper, or if you want it even faster, a conveyor inserter. Much of this seems incredibly large-scale to request as well, UAVs would be a huge task likely even more complex than trains, which are still incomplete after months of work.

**Reply by letovskiiy** [reply]
Thank you for your efforts. This is a wonderful mod!

**Comment by Tribun-2014** [comment]
Полностью согласен

**Reply by letovskiiy** [reply]
Спасибо!

**Comment by JameH2** [comment]
Suggestions arent allowed here sorry

**Reply by letovskiiy** [reply]
Why?
